.circleci/config.yml

```diff
@@ -29,6 +30,13 @@ executors:
 commands:
   bundle-yarn-install:
     steps:
+      - run:
+          name: Switch Node.js version
+          command: |
+            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
```

Comment:
:crying-cowboy-sunglasses: it's too bad even the [node orb in CircleCI](https://circleci.com/developer/orbs/orb/circleci/node) just pipes to bash too

Reply:
> :crying-cowboy-sunglasses: it's too bad even the [node orb in CircleCI](https://circleci.com/developer/orbs/orb/circleci/node) just pipes to bash too

Yeah, it seems to be the only official or recommended method to install 😬 

Also the same in this support article: https://support.circleci.com/hc/en-us/articles/360051656632-Swap-node-version-on-CircleCI-convenience-image